vscode/src/azure/commands.ts

```diff
@@ -21,13 +22,27 @@ import { QuantumUris, compileToBitcode } from "./networkRequests";
 import { getQirForActiveWindow } from "../qirGeneration";
 
 const corsDocsUri = "https://github.com/microsoft/qsharp/wiki/Enabling-CORS";
+const workspacesSecret = "qsharp-vscode.workspaces";
 
-export function initAzureWorkspaces() {
+export async function initAzureWorkspaces(context: vscode.ExtensionContext) {
```

Comment:
Should we add the return type now that this is async?

Reply:
It should still be inferred correctly, as it was before.